backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -98,6 +98,7 @@ public Person updatePatient(
         parseEmail(person.getEmail()),
         parseRace(person.getRace()),
         parseEthnicity(person.getEthnicity()),
+        person.getTribalAffiliation(),
```

Comment:
Looks like this is broken because the `PersonUpdate` model needs updating

Reply:
Thank you!